app/app.js

```diff
@@ -69,10 +69,9 @@ angular.module('arachne',[
 			'catalog':			{ url: '/catalog/:id?view', templateUrl: 'app/catalog/catalog.html', data: { pageTitle: title }},
 			'catalog.entry':	{ url: '/:entryId?view', templateUrl: 'app/catalog/catalog.html', data: { pageTitle: title }},
 			'books':			{ url: '/books/:id', templateUrl: 'app/entity/entity.html', reloadOnSearch: false, data: { pageTitle: title }},
-            'booksSuffixed':    { url: '/books/:id/:suffix?', templateUrl: 'app/entity/entity.html', reloadOnSearch: false, data: { pageTitle: title }},
+			'booksSuffixed':	{ url: '/books/:id/:suffix?', templateUrl: 'app/entity/entity.html', reloadOnSearch: false, data: { pageTitle: title }},
+			'booksSuffixedPage':	{ url: '/books/:id/:suffix/:page?', templateUrl: 'app/entity/entity.html', reloadOnSearch: false, data: { pageTitle: title }},
```

Comment:
Aus der Zukunft gedacht würde ich mir hier einen Verweis auf die Tickets wünschen, z.B.:

`// Temporary fix for WIT-185, also see Ticket SD-842`

Reply:
Ok, habs noch eingefügt.